apps/challenges/views.py

```diff
@@ -814,6 +814,46 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
             response_data = {"error": message}
             return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
+        if(data.get("submission_meta_attributes_schema")):  # To ensure that the schema for submission metaattributes is valid.
+            for attribute in data["submission_meta_attributes_schema"]:
+
+                try:
+                    name = attribute["name"]
+                except KeyError:
+                    message = (
+                    "No name found for attribute in challenge_phase {}".format(data["id"])
+                    )
+                    response_data = {"error": message}
+                    return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+                try:
+                    description = attribute["description"]
```

Comment:
@KhalidRmb can we create a generic method in utils which will take json object and `key` and will return whether that key exists or not. That way code will be clean and you won't need multiple `try except`